playbooks/fundamentals/openshift_roles_responsibilities.adoc

```diff
@@ -0,0 +1,235 @@
+- Platform security
+- Monitoring integration
+
+a|
+_Managing Tenant Provisioning, Isolation, and Capacity_
+
+Activities:
+
+- Adding users and teams to the platform
+- Design and manage quotas
+- Design and implement RBAC
+
+a|
+- Knowledge of Kubernetes and OpenShift architecture
+- Knowledge of containers and container architecture
+- Scripting and automation
+- Deep knowledge of projects, quotas, limits, roles, role bindings, and scheduling
+
+a|
+_Building and Maintaining Base Images_
+
+Activities:
+
+- Develop image change workflow
+- Develop standard base images
+
+a|
+- Linux system administration
+- Scripting and automation
+- Application and middleware runtime configuration
+- Knowledge of container architectures
+- Application build frameworks
+- Deep knowledge of images, imagestreams, templates
+
+a|
+_Design, Maintenance of Deployment Pipeline_
+
+Activities:
+
+- Design and document pipeline standards
+- Develop quickstarts and templates
+- Educate development teams
+
+a|
+- Source code management
+- Application design and implementation
+- Scripting and automation
+- Automated testing
+- Code quality testing
+- Knowledge of container architectures
+- Knowledge of immutable infrastructure
+- Security - access control of pipeline steps, proper approval workflows etc.
+- Deep knowledge of OpenShift templates, buildconfigs, deploymentconfigs, services, routes, configmaps
+
+a|
+_Application/Test development_
+
+Activities:
+
+- Application coding
+- Automated test development
+- Responding to deployment pipeline test failures
+- Responding to application outages
+- User acceptance testing
+
+a|
+- Application design and implementation
+- Automated testing
+- Source code management
+- Application monitoring
+- Knowledge of cloud native application architectures
+
+a|
+_Monitoring and Managing Application Operational Behavior_
+
+Activities:
+
+- Design application for performance
+- Monitor runtime behavior of the application
+- Scale applications (or autoscale)
+- Manage application availability
+- Request quotas and manage resource limits
+- Performance and capacity test
+
+a|
+- Application performance design and implementation
+- Application performance monitoring
+- Performance and load testing
+
+a|
+_User Acceptance Testing_
+
+Activities:
+
+- Testing UIs for design and human interaction
+- Developing automated tests
+
+a|
+- Human/interface design/validation
+- Automated testing patterns
+- Testing frameworks
+- Application design patterns
+
+|===
+
+== Roles for IT Organizations Using OpenShift
+
+As organizations shift toward a more DevOps oriented organizational model, the number of specialized roles may decrease, as cross functional teams and roles increase to maximize collaboration opportunities. These are the core roles we see as relevant in an IT organization using OpenShift.
+
+- **Application Operations Engineer OR Site Reliability Engineer.** May have been formerly Application Server Administrators.
```

Comment:
Swap words to match description below "May have formally been a" and correct spelling of _formally_